pilot/pkg/model/push_context.go

```diff
@@ -829,10 +841,12 @@ func (ps *PushContext) updateContext(
 
 	if authnChanged {
 		if err := ps.initAuthnPolicies(env); err != nil {
+			log.Errorf("failed to initialize authn policies %v", err)
```

Comment:
nit: do we not log the err later? should we just wrap the error with context

Reply:
Removed. Return err should be good enough.